service/assessment/assessment.go

```diff
@@ -250,19 +250,17 @@ func (svc *Service) AssessEvidence(_ context.Context, req *assessment.AssessEvid
 	if err != nil {
 		err = fmt.Errorf("error while handling evidence: %v", err)
 		log.Error(err)
-		return res, status.Errorf(codes.Internal, "%v", err)
+		return nil, status.Errorf(codes.Internal, "%v", err)
 	}
 
-	res = &assessment.AssessEvidenceResponse{}
-
-	return res, nil
+	return nil, nil
```

Comment:
Shouldn't we return the empty response here? This was ok I think. Or is this automatically converted into an "empty" AssessEvidenceResponse?

Reply:
That is a good question. I thought that since we don't need the AssessEvidenceResponse, we might as well set it to nil. But you are right, maybe it is better to return an empty response for future changes. Otherwise we have to change the tests again.